Project 1/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
```

Comment:
The formatting on this file is very inconsistent; did you install and enable the [Beautify](https://marketplace.visualstudio.com/items?itemName=HookyQR.beautify) plugin for VS Code?